src/borg/archive.py

```diff
@@ -86,7 +87,7 @@ def show_progress(self, item=None, final=False, stream=None, dt=None):
             columns, lines = get_terminal_size()
             if not final:
                 msg = '{0.osize_fmt} O {0.csize_fmt} C {0.usize_fmt} D {0.nfiles} N '.format(self)
```

Comment:
i think this would be more awesome, if __format__ was implemented, and the attribute names where combined with getattr